application/models/card.py

```diff
@@ -5,8 +5,8 @@ class Card(db.Model):
     __tablename__ = 'card'
     name = db.Column(db.String(50), primary_key=True)
     noPlayers = db.Column(db.Integer, primary_key=True)
-    age = db.Column(db.Integer, default=1)
-    type = db.Column(db.String(10))
+    age = db.Column(db.Integer, primary_key=True)
```

Comment:
noPlayers: represents the amount of players needed for that card to be included. A card may be entered in the database multiple times with different player counts in order to allow for multiple copies of a card when there's lots of players.

Age: 3 cards (the greys) are repeated in age 1 and 2 with all other attributes being equal.